docs/usage/development/updating.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Updating
+---
+import Alert from '@theme/Alert'
+
+<Alert title="Please note" type="warning" icon="alert">
+    Especially during the alpha and beta phases, you are expected to keep all tauri dependencies and toolchains up to date. There is no support for any versions other than latest.
+</Alert>
+
+## Current Latest Versions
+| Component | Description  | Version |
+|-----------|---------|------|
+| [**tauri.js**](https://github.com/tauri-apps/tauri/tree/dev/cli/tauri.js) | CLI and imports | [![](https://img.shields.io/npm/v/tauri.svg)](https://www.npmjs.com/package/tauri) |
+| [**tauri core**](https://github.com/tauri-apps/tauri/tree/dev/tauri) | rust API and tooling | [![](https://img.shields.io/crates/v/tauri.svg)](https://crates.io/crates/tauri)|
+| [**tauri bundler**](https://github.com/tauri-apps/tauri/tree/dev/cli/tauri-bundler) | binary bundler | [![](https://img.shields.io/crates/v/tauri-bundler.svg)](https://crates.io/crates/tauri-bundler)  |
+
+
+## Update NPM Packages
+
+If you are using the `tauri` package:
+```bash
+$ yarn upgrade tauri --latest
+$ npm install tauri@latest
+```
+
+Alternatively, if you are using the `vue-cli-plugin-tauri` approach:
+```bash
+$ yarn upgrade vue-cli-plugin-tauri --latest
+$ npm install vue-cli-plugin-tauri@latest
+```
+
+## Update Cargo Packages
+Go to `src-tauri/Cargo.toml` and change `tauri` to
+`tauri = { version = "%version%" }` where `%version%` is the version number shown above. (You can just use the `MAJOR.MINOR`) version, like `0.9`.
+
+Then do the following:
+```bash
+$ cd src-tauri
+$ cargo update -p tauri
+```
+
```

Comment:
It's not part of cargo itself so it may be out of scope, but it may be worth mentioning `cargo-outdated` to help check for outdated packages.  Usage in this case would be `cargo outdated -R` which would give output like
```ps
Name   Project  Compat  Latest  Kind    Platform
----   -------  ------  ------  ----    --------
tauri  0.7.4    0.7.5   0.7.5   Normal  ---
```

Reply:
`cargo outdated -p tauri` can also be used instead of the `-R` flag (shows only root packages) to only specifically check `tauri` and not show other outdated root packages